internal/utils/misc.go

```diff
@@ -27,25 +27,27 @@ const (
 	Pg13Image = "supabase/postgres:13.3.0"
 	Pg14Image = "supabase/postgres:14.1.0.89"
 	// Append to ServiceImages when adding new dependencies below
-	KongImage      = "library/kong:2.8.1"
-	InbucketImage  = "inbucket/inbucket:3.0.3"
-	PostgrestImage = "postgrest/postgrest:v9.0.1.20220717"
-	DifferImage    = "supabase/pgadmin-schema-diff:cli-0.0.5"
-	MigraImage     = "djrobstep/migra:3.0.1621480950"
-	PgmetaImage    = "supabase/postgres-meta:v0.53.1"
-	StudioImage    = "supabase/studio:0.22.08"
-	DenoRelayImage = "supabase/deno-relay:v1.5.0"
+	KongImage       = "library/kong:2.8.1"
+	InbucketImage   = "inbucket/inbucket:3.0.3"
+	PostgrestImage  = "postgrest/postgrest:v9.0.1.20220717"
+	DifferImage     = "supabase/pgadmin-schema-diff:cli-0.0.5"
+	MigraImage      = "djrobstep/migra:3.0.1621480950"
+	PgmetaImage     = "supabase/postgres-meta:v0.53.1"
+	StudioImage     = "supabase/studio:0.22.08"
+	DenoRelayImage  = "supabase/deno-relay:v1.5.0"
+	ImageProxyImage = "supabase/imgproxy:v1.0.2"
```

Comment:
Could you point me to the repo where this image is built from? Need to setup ecr and ghcr mirrors.

Reply:
I already added ecr support Qiao - https://github.com/supabase/imgproxy/

Reply:
mirrored to ghcr too and fixed tests

Reply:
Nice! Thank you